content/docs/infrastructure-wg.md

```diff
@@ -0,0 +1,168 @@
+---
+title: Infrastructure Working Group
+bookToc: true
+---
+# Infrastructure Working Group (in the course of formation)
+The infrastructure working group (IWG) understands the needs and pains of the 
+community and shapes the project infrastructure to meet these needs. 
+
+## Scope
+
+The working group will work on the LLVM project infrastructure including:
+
+* source code repository
+* bug tracker
+* build servers/machines
+* web sites
+* build system/scripts
+* code review tool
+* mailing lists
+* internet forum
+* chat
+* Documentation of the tools and their work flows.
+
+
+## Working group members
+
+The focus of IWG members will be on identifying, designing and agreeing on 
+infrastructure improvements. The implementation of the changes and the 
+operations of our infrastructure can be offloaded to community members or 
+contractors. This way we keep the workload for the working group low.
+
+Working group members should have the oppurtunity to dedicate 4-8 hours per 
+week on a regular basis in the initial phase, as we expect to have a significant 
+pile of decisions to take. It should be less than 4 hours once the roadmap has 
+become more stable.
+
+We would like to have broad and diverse community representation within the 
+working group as we are convinced such groups deliver the best results. Please
+contact us if you want to participate.
+
+Initial members:
+
+* Christian Kühnel
+* ... we're recruting plese contact us at TODO!
```

Comment:
Tanya should have an email alias setup in the next day or so.  We should update this with the address before we publish.